explorer/app/config/model.ts

```diff
@@ -1,5 +1,34 @@
+import {
+  DetailResponseType,
+  DetailViewModel,
+  ListResponseType,
+  ListViewModel,
+} from "app/models";
 import { HeaderProps } from "../components/Header/Header";
```

Comment:
I have been reviewing the barrel functionality and for consistency with our other projects, would you be able to remove all barrels (here and throughout) and instead import directly from where values are declared?